doc/releases/migration-guide-4.2.rst

```diff
@@ -299,6 +299,13 @@ SPI
 Other subsystems
 ****************
 
+LoRa
+====
+
+* The :c:member:`lorawan_downlink_cb.cb` callback is now passed a pointer to the parent
+  :c:struct:`lorawan_downlink_cb` instance as the first parameter. This callback pointer can be used
+  to reference outer user-defined data structure using ``CONTAINER_OF`` macro.
```

Comment:
```suggestion
  to reference outer user-defined data structure using :c:macro:`CONTAINER_OF` macro.
```

Reply:
I know the current text was verbatim my suggestion, sorry

Reply:
Improvements are always welcome. Applied, thanks.